numpy/linalg/linalg.py

```diff
@@ -2639,6 +2639,13 @@ def multi_dot(arrays): return functools.reduce(np.dot, arrays)
         If the first argument is 1-D it is treated as row vector.
         If the last argument is 1-D it is treated as column vector.
         The other arguments must be 2-D.
+    out : ndarray, optional
+        Output argument. This must have the exact kind that would be returned
+        if it was not used. In particular, it must have the right type, must be
+        C-contiguous, and its dtype must be the dtype that would be returned
+        for `dot(a,b)`. This is a performance feature. Therefore, if these
```

Comment:
```suggestion
        for ``dot(a, b)``. This is a performance feature. Therefore, if these
```

Reply:
added in 2a4c4c9c5b667b3d67ca1cc9bfd7b80d5a59e6d1